test/helpers/serviceBuilders.js

```diff
@@ -53,3 +53,7 @@ export function buildTestSmartContractService() {
 export function buildTestEventService() {
   return buildTestService('event', { event: true });
 }
+
+export function buildTestNonceService() {
```

Comment:
not a big deal, but if you're only going to use this in a single test file you might as well just import `buildTestService` into that file and call the line below there